hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/host/HostsFacade.java

```diff
@@ -166,4 +163,76 @@ public Long totalMemoryCapacity() {
     TypedQuery<Long> query = em.createNamedQuery("Host.TotalMemoryCapacity", Long.class);
     return query.getSingleResult();
   }
+  
+  public enum Sorts {
+    HOSTNAME("HOSTNAME", "LOWER(h.hostname)", "ASC");
```

Comment:
@kai-chi can we sort by more fields, like cores, gpus, memory, last hearbeat etc?